web/src/components/netflow-traffic-parent.tsx

```diff
@@ -49,7 +49,7 @@ class NetflowTrafficParent extends React.Component<Props, State> {
       return this.props.children;
     }
     // else render default NetworkTraffic
-    return <NetflowTraffic />;
+    return <NetflowTraffic forcedFilters={null} />;
```

Comment:
why do we need a distinction null vs undefined?

Reply:
Coming from `netflow-tab`, the prop will be undefined before being set.

As we force null in `netflow-traffic-parent`, we make the distinction between the loading state from tabs and the null state from page to [skip a tick before confing loaded](https://github.com/netobserv/network-observability-console-plugin/pull/273/files#diff-aeef245327564ecae089389ec6a41a108f749f48003cde41d9f3095da8e09464R455-R458)